Updated rebrand colours

amymccorry opened the pull request:
Replaced the 3 original palettes with the new rebrand colours as 6 RGB, 6 darker RGB and 6 monochrome colours. Updated the `exploristics_colour` and `exploristics_fill` functions to use the new colours. Replaced the old logo with the new, and added the option to apply a white or navy version. Updated the advanced use vignette with example plots using the new palettes and logo. Fixed an issue in which `text_wrapper` was not functioning properly.

amymccorry committed:
Colour pallettes updated. Company logo updated with dark and light versions.

amymccorry committed:
Updated the vignette examples with plots using the new colours

amymccorry committed:
Fixed issue with `text_wrapper()` not applying the processed labels correctly to plots

GABurns commented on R/data.R:
```diff
@@ -1,24 +1,14 @@
+)
+Expl_Mono <- c(
+  "#121a31", "#16213d", "#2d3750", "#5c6477", "#ecf1f8", "#ffffff"
+)
```
The data.R file is a special case for the documentation of package data, rather than the place to put data. Not sure why it's set up like this but took me a while to figure out as the documentation of how to document data wasn't very clear.

So I'll fix this so devtools::check doesn't return any warnings for undocumented data

GABurns commented on R/exploristics_colour.R:
```diff
@@ -73,29 +72,29 @@ exploristics_colour <-
+            plot + scale_color_gradientn(colours = rev(ExploristicsTheme::Expl_RGB_dark[c(1, 4, 2)]))
         }
-        if (colourPalette == "Expl_HighCont") {
+        if (colourPalette == "Expl_Mono") {
           plot <-
-            plot + scale_color_gradientn(colours = rev(ExploristicsTheme::Expl_HighCont[c(1, 2, 4)]))
+            plot + scale_color_gradientn(colours = rev(ExploristicsTheme::Expl_Mono[c(5, 3, 2)]))
         }
       } else {
         # continuous
-        if (colourPalette == "Expl_Blue") {
-          plot <- plot + scale_color_gradientn(colours = ExploristicsTheme::Expl_Blue)
+        if (colourPalette == "Expl_RGB") {
+          plot <- plot + scale_color_gradientn(colours = ExploristicsTheme::Expl_RGB[c(1, 4, 2)])
         }
-        if (colourPalette == "Expl_External") {
+        if (colourPalette == "Expl_RGB_dark") {
           plot <-
-            plot + scale_color_gradientn(colours = ExploristicsTheme::Expl_External[c(1, 6, 2)])
+            plot + scale_color_gradientn(colours = ExploristicsTheme::Expl_RGB_dark[c(1, 4, 2)])
         }
-        if (colourPalette == "Expl_HighCont") {
+        if (colourPalette == "Expl_Mono") {
           plot <-
```
Nested if else statements will flag up a warning in the code quality - it's known as cyclomatic complexity. 

I'm happy to leave it in - if you've not used it before the `switch` statement in R is similar to `case_when` from dplyr. It makes code easier to follow and maintain.

GABurns commented on R/save_with_logo.R:
```diff
@@ -80,7 +81,8 @@ save_with_logo <-
            height = 8,
            dpi = 300,
-           suffix = NULL) {
+           suffix = NULL,
```
The examples still contain the old colour palettes. I've updated them to Expl_RGB

GABurns committed:
Resolving Merge Conflicts and recompling documentation

Merge branch 'main' into updated_rebrand_colours

# Conflicts:
#	R/data.R
#	data-raw/DATASET.R

GABurns reviewed (dismissed):
I updated a the Readme file as it still had some references to old colour schemes and also within some examples. 

Needed to update some unit tests that were failing also
